zenoh/src/net/routing/dispatcher/face.rs

```diff
@@ -212,57 +215,17 @@ impl Primitives for Face {
     fn send_interest(&self, msg: zenoh_protocol::network::Interest) {
         let ctrl_lock = zlock!(self.tables.ctrl_lock);
         if msg.mode != InterestMode::Final {
```

Comment:
Maybe change this to `msg.mode.final()` and invert the branches?

```suggestion
        if msg.mode.final() {
```

Reply:
Yes there is no `final` function (we can add it easily anyway...). I still think it easier to read if the branches are inverted. This is mostly a nitpick.